lib/cisco_node_utils/cmd_ref/interface_channel_group.yaml

```diff
@@ -16,18 +16,34 @@ channel_group:
   kind: int
   default_value: false
   nexus:
-    get_value: '/^channel-group (\d+)$/'
+    get_value: '/^channel-group (\d+)\s?\S*\s?\S*$/'
```

Comment:
Fair enough.  You should be able to match either case by simply removing the trailing anchor `$`.

```ruby
get_value: '/^channel-group (\d+)/'
```

```ruby
2.2.1 :010 >   line = 'channel-group 5'
 => "channel-group 5" 
2.2.1 :011 > /^channel-group (\d+)/.match(line)
 => #<MatchData "channel-group 5" 1:"5"> 
2.2.1 :012 > 
2.2.1 :013 >   line = 'channel-group 5 mode active'
 => "channel-group 5 mode active" 
2.2.1 :014 > 
2.2.1 :015 >   /^channel-group (\d+)/.match(line)
 => #<MatchData "channel-group 5" 1:"5"> 
2.2.1 :016 > 
```